container-images/build_llama_and_whisper.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
+
+# Bash does not easily pass arrays as a single arg to a function.  So, make this a global var in the script.
+CMAKE_FLAGS=""
+LLAMA_CPP_CMAKE_FLAGS=""
```

Comment:
If we need a global variables, could we put local variables in main instead? Trying to keep everything scoped.

Reply:
The styling changes aren't a blocker for merge, but why change it?

We don't put "function" in from of the functions in any of the other shell scripts in the project, we don't use camelCase in any other shell scripts either.

Now this is the odd shell script in terms of consistency 😄

Reply:
Yeah, I don't want to be the oddball...